pageserver/src/tenant/upload_queue.rs

```diff
@@ -58,7 +57,11 @@ pub(crate) struct UploadQueueInitialized {
     /// uploaded. `Lsn(0)` if nothing was uploaded yet.
     /// Unlike `latest_files` or `latest_metadata`, this value is never ahead.
     /// Safekeeper can rely on it to make decisions for WAL storage.
-    pub(crate) last_uploaded_consistent_lsn: Lsn,
+    ///
+    /// visible_remote_consistent_lsn is only updated after our generation has been validated with
+    /// the control plane.
```

Comment:
That `store` is just the legacy case that'll get removed -- I'm mostly not caveating all the comments about generations with the "...but in legacy mode..." case since it'll go away soon, but I've extended this comment to be more literally accurate by mentioning the legacy case.